h2o-py/h2o/frame.py

```diff
@@ -414,16 +413,16 @@ def show(self, use_pandas=False):
         if self._ex is None:
             print("This H2OFrame has been removed.")
             return
-        if not self._ex._cache.is_valid(): self._frame()._ex._cache.fill()
+        if not self._ex._cache.is_valid(): self._frame()._ex._cache.fill(rows=self.nrows)
```

Comment:
but still, doing `cache.fill(rows=self.nrows)` is not good in any environment. It will lead to crash of the client in case of big data.